src/game/playt.mts

```diff
@@ -96,15 +96,5 @@ export function surrender(score: number) {
 }
 
 export function endTutorial() {
-  return fetch(`${process.env.API_HOST}/api/tutorials/scores`, {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json",
-    },
-    body: JSON.stringify({
-      playerToken,
-      score: 0,
-      finalSnapshot: true,
-    }),
-  });
+  return submitScore(0);
```

Comment:
Given the recent `tutorial` query param discussion, I hope this is also what every FS game does otherwise we've got a security issue ^^

Reply:
Ya, "hope" ...